sourcegraph/sourcegraph
# Code Insights: [FE] Connect capture group creation page to api

Closes https://github.com/sourcegraph/sourcegraph/issues/28026

This PR is the first PR of three for finishing capture group creation UI
- This one
- https://github.com/sourcegraph/sourcegraph/pull/28855
- https://github.com/sourcegraph/sourcegraph/pull/28861

This PR does a few things 
- Updates GQL API (supports capture group insight for creating/updating mutations)
- Adds capture group insight interface 
- Connects the capture group insight page to GQL API

## How to test
- Go to /insights/create/capture-group
- Fill in the form and click create insight
- See in the console that network request with create mutation has appeared in the network tab
- Check your "All insights dashboard"
